x/profiles/spec/02-concepts.md

```diff
@@ -119,6 +119,9 @@ This will start an interactive prompt session allowing you to generate the prope
    This will effectively link your Desmos profile to the external chain address. The required argument is the (absolute) path to the file generated using the `create-chain-link-json` command.
 
 
+## Default External Address
+A default external address represents a default chain link which makes dealing with a single chain link per chain for a profile easier. Each chain must have a chain link as a default external address for interaction, it will pick up the oldest chain link as a default external address if the link of the default external address is unlinked.
```

Comment:
```suggestion
Since each user can have multiple links to the same chain, we allow users to specify which address should be considered as the default one for each chain. By default, the first link created for each chain will be considered as the default one. Also, if a chain link was set as the default one and is later deleted, the oldest chain link for the same chain (if any) will be used as the default one.
```